### ISSUE TITLE
regression: 'starbucks 10010'

### ISSUE THREAD

**Issue by missinglink** [pelias/api#425]
regression **on dev** for `/v1/search?text=starbucks 10010`.

```
# prod

 1)	Starbucks, Manhattan, NY
 ...

#dev

 1)	100 East 22nd Street, Manhattan, New York
 2)	100 East 24th Street, Manhattan, New York
 3)	100 East 25th Street, Manhattan, New York
 4)	100 East 25 Street, Manhattan, New York
 5)	100 East 24 Street, Manhattan, New York
 6)	100 East 22 Street, Manhattan, New York
 7)	STARIN 430, Buffalo, New York
 8)	7 Strada Statale, Piverone, Italy
 9)	10 East 22 Street, Manhattan, New York
10)	10 West 26th Street, Manhattan, New York
```

The document seems to be imported correctly to dev, it's just not being matched and returned correctly!?

- http://pelias.github.io/compare/#/v1/place%3Fids=openstreetmap:venue:node:2300348743
- http://pelias.github.io/compare/#/v1/place%3Fids=osm:venue:2300348743

**Comment by missinglink** [comment]
```
$ git diff wof_focus_weighting master -- test/unit/fixture/search_*

$ echo $?
0
```

```
git diff wof_focus_weighting master -- query/search_defaults.js

$ echo $?
0
```

```diff
$ git diff wof_focus_weighting master -- query/search.js
diff --git a/query/search.js b/query/search.js
index d75e3f4..764cf99 100644
--- a/query/search.js
+++ b/query/search.js
@@ -37,7 +37,7 @@ query.score( peliasQuery.view.admin('neighborhood') );
 // non-scoring hard filters
 query.filter( peliasQuery.view.boundary_circle );
 query.filter( peliasQuery.view.boundary_rect );
-query.filter( peliasQuery.view.sources );
+
 // --------------------------------
 
 /**
@@ -51,8 +51,6 @@ function generateQuery( clean ){
   // input text
   vs.var( 'input:name', clean.text );
 
-  vs.var( 'sources', clean.sources);
-
   // size
   if( clean.querySize ) {
     vs.var( 'size', clean.querySize );
@@ -121,9 +119,7 @@ function generateQuery( clean ){
     textParser( clean.parsed_text, vs );
   }
 
-  var q = query.render( vs );
-  //console.log(JSON.stringify(q, null, 2));
-  return q;
+  return query.render( vs );
 }
 
 // return diagonal distance in km, with min=1
```

**Comment by missinglink** [comment]
oops, I deleted that record on dev by accident. will wait for a rebuild to investigate further.

for some reason a simple query such as this doesn't even return the record in question:

```
GET /pelias/_search
{
   "query": {
      "match": {
         "phrase.default": {
            "analyzer": "standard",
            "query": "starbuck"
         }
      }
   },
   "filter": {
       "term": {
           "address.zip": "10010"
       }
   }
}
```

I checked the analysis again and they all look fine, I suspect that maybe the `phrase.*` info is not being imported for OSM, however this query returns 227674948 results:

```
GET /pelias/_search
{
   "filter": {
       "exists": {
           "field": "phrase.default"
       }
   }
}
```

**Comment by orangejulius** [comment]
We fixed this! :tada: I believe it was fixed by pelias/pelias#278
